src/components/Breadcrumbs.js

```diff
@@ -22,10 +22,17 @@ export default class Breadcrumbs extends Component {
       const paths = splat.split('/');
       links = _.map(paths, (path, i) => {
         const before = (i == 0) ? '' : paths.slice(0, i).join('/') + '/';
-        return {
-          href: `${base}/${before}${path}`,
-          label: path
-        };
+        if (type == "data files") {
```

Comment:
are you referring to the pre-API variable or `path` within `Breadcrumbs`? If its the latter, the reason is that variable `before` uses `path` to generate its value and I get the following:
```
Data Files / movies/ / genres/ /
```